test/karma/test-app/slot-fallback/karma.spec.ts

```diff
@@ -1,7 +1,8 @@
 import { setupDomTests, waitForChanges } from '../util';
 
 
-describe('slot-fallback', () => {
+// TODO(STENCIL-18) Restore this test and fix the underlying issue.
+xdescribe('slot-fallback', () => {
```

Comment:
FWIW I normally prefer the `.skip` syntax here, but we're using Jasmine here, which doesn't have that in it's API